src/Broker/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+project(Broker VERSION 1.0.0 LANGUAGES C)
+
+add_library(Broker STATIC)
+target_sources(Broker
+    PRIVATE     Broker.c
+    PUBLIC      Broker.h)
+
+target_include_directories(Broker
+  PUBLIC      ${CMAKE_CURRENT_SOURCE_DIR})
+
+target_link_libraries(Broker
+  PRIVATE     common_build_options)
+
+target_link_libraries(Broker
+  PUBLIC      TASTE::Mocks)
+
+target_link_libraries(Broker
```

Comment:
All libraries can/should be listed in one `target_link_libraries` call

Reply:
fixed